test/controllers/import.test.js

```diff
@@ -666,4 +669,101 @@ describe('ImportController tests', () => {
       expect(mockDataAccess.removeImportJob.getCall(0).args[0].getId()).to.equal(exampleJob.id);
     });
   });
+
+  describe('stopImportJob', () => {
+    it('should fail when api key does not have the correct scopes', async () => {
+      baseContext.auth.checkScopes = sandbox.stub().throws(new Error('Invalid scopes'));
```

Comment:
Ah, so it is called. But which scopes are being checked for this feature?

Reply:
we look for the imports.write scope for stopping an import job.